src/main/java/org/mongen/core/MongenCoreService.java

```diff
@@ -10,4 +14,15 @@ public static void main(String[] args) {
 		SpringApplication.run(MongenCoreService.class, args);
 	}
 
+	@Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurerAdapter() {
+			@Override
+			public void addCorsMappings(CorsRegistry registry) {
+				registry.addMapping("/api/v1/countries").allowedOrigins("http://localhost:3000");
+				registry.addMapping("/api/v1/collaborators/type").allowedOrigins("http://localhost:3000");
+			}
```

Comment:
should we add more stuff here?

Reply:
Yeah, I will take to see if we can apply this CORS policy to all the endpoints